nemo/collections/nlp/models/language_modeling/megatron_lm_encoder_decoder_model.py

```diff
@@ -1415,3 +1411,17 @@ def _validate_trainer(self):
 
     def list_available_models(self):
         pass
+
+    def build_model_parallel_config(self):
+        """ Hidden size needs to be set from the cfg.encoder for the pipeline schedule.
+        """
+
+        model_parallel_config = super().build_model_parallel_config()
```

Comment:
Wouldn't parent class return warning if `hidden_size` is not in `cfg.model.hidden_size`? Perhaps this argument can be passed to parent method?

Reply:
Maybe you could expand more on your suggestion? I was adding this because the parent class didn't have hidden_size for this model.